src/pages/layout/css-utilities.md

```diff
@@ -176,6 +176,44 @@ The table below shows the default behavior, where `{modifier}` is any of the fol
 | `float-lg-{modifier}`  | Applies the modifier to the element when `min-width: 992px`.        |
 | `float-xl-{modifier}`  | Applies the modifier to the element when `min-width: 1200px`.       |
 
+## Element Display
+
+The display CSS property determines if an element should be visible or not. The element will still be in the DOM, but not rendered, if it is hidden.
+
+```html
+<ion-grid>
+  <ion-row>
+    <ion-col class="ion-hide">
+      <div>
+        <h3>hidden</h3>
+        You can't see me.
+      </div>
+    </ion-col>
+    <ion-col>
+      <div>
+        <h3>not-hidden</h3>
+        You can see me!
+      </div>
+    </ion-col>
+  </ion-row>
+</ion-grid>
+```
+
+| Class         | Style Rule                      | Description                                                                                               |
+|---------------|---------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `.ion-hide`   | `display: none`                 | The element will be hidden. 
+
+### Responsive Display Attributes
+
+There are also additional classes to modify the visibility based on the screen size. Instead of just `.ion-hide` as a sole class, use `.ion-hide-{breakpoint}-{dir}` to only use the attribute on specific screen sizes, where `{breakpoint}` is one of the defined breakpoints in the `--screen-breakpoints` variable, and `{dir}` is whether the element should be hidden on all screen sizes above (`up`) or below (`down`) the current one.
+The table below shows the default behavior, where `{dir}` is any of the following: `up` or `down`, as they are described above.
```

Comment:
```suggestion

```

Reply:
Cool! Apparently github offers the ability to just commit suggestions. Definitely, I think that phrasing is redundant with the paragraph above it after looking further at it.